### PR TITLE
Add info about route optimized field

### PR DESCRIPTION
**Pull request by loganakamatsu**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/models/optimized.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ each will be returned. You will never need to create these Optimized models. The
 | startTime                 | DateTime                                       | false    | The estimated start time of the route.
 | name                      | String                                         | false    | The name of the route.
 | waypoints                 | List<[OptimizedWaypoint](#optimizedwaypoint)>  | false    | The list of waypoint objects. This list will not be empty.
-| startingWarehouse         | [OptimizedWarehouse](#optimizedwarehouse)     | false    | The starting point for the route.
-| endingWarehouse           | [OptimizedWarehouse](#optimizedwarehouse)     | false    | The endoing point for the route.
+| startingWarehouse         | [OptimizedWarehouse](#optimizedwarehouse)      | false    | The starting point for the route.
+| endingWarehouse           | [OptimizedWarehouse](#optimizedwarehouse)      | false    | The endoing point for the route.
```
Comment (hmvien) [inline source/includes/models/optimized.md]: `The ending point for the route.`

Comment (loganakamatsu) [reply source/includes/models/optimized.md]: nice

Comment (loganakamatsu) [reply source/includes/models/optimized.md]: Fixed in 1c04c2b

**Comment by hmvien** [comment]
:shipit:

**Comment by loganakamatsu** [comment]
Thanks man.

**Pull request merged**